### DISCUSSION TITLE
Last release two years ago - why no regular releases?

### DISCUSSION THREAD

**Discussion by philipmw** [emacs-lsp/lsp-mode#4268] [General]
Hello all. I am a new Emacs user, exploring using Emacs as an IDE for a TypeScript-based project. I ran into an issue that I believe was fixed two months ago: https://github.com/emacs-lsp/lsp-mode/pull/4202

The issue was fixed in the codebase, but I have no clear way to get this change. My Emacs installed packages are all on their latest versions. `lsp-mode` is on version 8.0.0, which is both the latest available version and is two years old.

Now friends suggest I use a different package system to pick up a newer git commit. I could do that, but that adds complexity to my Emacs setup. I am concerned on behalf of all *lsp-mode* users who might run into any issue discovered and fixed over the last two years, have no simple way to get the fix, and maybe give up on *lsp-mode* and even *Emacs* entirely.

My ideal outcome is that *lsp-mode* would make a new release regularly and frequently, and my Emacs would update to it. Why hasn't there been a release in over two years? That is unusual for a software project.

**edit:** A friend pointed out that I am using *MELPA Stable*, while the non-stable MELPA has a much newer version. That's my path forward for now. Still, I believe the issue is relevant since we know that the MELPA Stable version is broken, yet the project doesn't make a release.

**Comment by yyoncho** [comment]
I will write more detailed answer later but as a starter you may check this one: https://www.reddit.com/r/emacs/comments/etikbz/speaking_as_a_package_maintainer_please_do_not/

**Comment by philipmw** [comment]
Thanks; this is very helpful. As a new Emacs user, I didn't know about the bigger picture and overall shortcoming of MELPA Stable.
